docs/design/kademlia-removal.md

```diff
@@ -0,0 +1,89 @@
+locating peer nodes, exchanging messages and sharing data. However, due to the nature of our network, we only use Kademlia 
+for node discovery and address lookups given node IDs. This is useful when satellites don’t know about all the nodes in 
+the network and nodes are unfamiliar with all of the satellites in the network.
+
+With our recent business decision of simplifying the storage node operator user experience, we no longer require kademlia 
+for node discovery. In a solution called SNO-select, storage nodes operators manually select the satellites they want to work with 
+and satellites wait for storage nodes to work with them. The initial implementation of this solution allows SNOs to update 
+their trusted satellite list in a hardcoded configuration file, but future improvements will enable users to manage
+this list through a web console. 
+
+We will replace our Kademlia DHT and related entities with direct communication between satellites and storage nodes, 
+and keep the network fresh without kademlia node discovery and random lookups.
+
+## Design
+
+### Nodes reach out to satellites that they want to work with
+- The satellites are listed in the trust package
+- Nodes should communicate with satellites directly rather than using kademlia to traverse the network to find the address of a given ID.
+- Storage nodes should notify satellites when they start up, wait a random amount of time (to add jitter 
+http://highscalability.com/blog/2012/4/17/youtube-strategy-adding-jitter-isnt-a-bug.html), then start reporting in roughly on the hour
+
+### Network refreshes at a regular interval
+- Nodes will keep themselves up to date in the network by pinging all the satellites in their
+   trusted list every hour.
+- Satellites will ping the nodes back to confirm their addresses
+    - If is it successful, the satellite will insert or update the node in the overlay cache and
+       notify the node of success. Make sure to close the connection. Don’t use the transport observer to update the cache.
+       Update the IP and uptime directly.
+    - If the satellite does not confirm the node address, it does not proceed with updating the overlay cache. The node 
+    receives a log message and closes the connection when it times out.
+
+### Disintegrate Kademlia from the network, storj sim and testplanet setups
+- Remove kademlia from the discovery package
+- Remove the bootstrap node - work with Ops
+  - Remove the vouchers service and related tables
+  - Work with QA to make sure storage nodes don’t crash on errors related to the elimination of Kademlia
+- if they don’t update immediately ->  keep just the overlay.Ping rpc method, it will be much easier for a new satellite 
+to work with old and new storage nodes.
+
+### Update whitepaper to address kademlia removal and the addition of satellite opt-in
+  - Delete the audit gating design doc
```

Comment:
sorry im late to the CR party, buy I'm wondering if we want to create an `docs/archive` directory and move old not-no-longer-relevant design docs there instead of deleting?

Reply:
we could do that!